app/controllers/platform_admin/photos_controller.rb

```diff
@@ -5,6 +5,7 @@ def index
 
   def edit
     @photo = Photo.find(params[:id])
+    @categories = Category.all
```

Comment:
Use all_categories from the application controller directly in the view instead.